src/Components/ItemInfo/index.jsx

```diff
@@ -0,0 +1,29 @@
+function ItemInfo({ itemName, itemValue }) {
+  let backgroundColour;
+  let textColour;
+  let textContent;
+
+if (itemName == "calories"){
+    backgroundColour = "bg-cyan-500";
+    textColour = "text-white";
+    textContent = `Calories: ${itemValue}`;
+} else if (itemName == "foodType"){
+    backgroundColour = "bg-blue-500";
+    textColour = "text-white";
+    textContent = `Type: ${itemValue}`;
+} else if (itemName == "sideItem"){
+    backgroundColour = "bg-yellow-400";
+    textColour = "text-black";
+    textContent = "Side";
+}
```

Comment:
Indentation / formatting. No excuses for formatting being bad when you can just press Prettier >:(

Reply:
Done :)